README.md

```diff
@@ -55,3 +55,9 @@ You should see output similar to the following:
  * Debugger PIN: 226-556-590
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+## Running Tests
+
+To set up the virtual environment, press `Cmd/Ctrl + Shift + P` and select `Python: Select Interpreter`. Select the Python executable in the new `.venv` directory, which is `./.venv/Scripts/python.exe` on Windows.
```

Comment:
Are you sure this is necessary?

My understanding is that pretending `poetry run` to the `pytest` command will mean the tests run in the context of the virtual environment managed by poetry.

Reply:
I think that is just if you want to run the tests in VS code, but it's what I did